ncm-ccm/src/main/pan/components/ccm/schema.pan

```diff
@@ -7,6 +7,7 @@ declaration template components/ccm/schema;
 
 include 'quattor/types/component';
 include 'pan/types';
+include 'components/accounts/functions';
```

Comment:
There definitely feels like potential for a bootstrap problem here. I guess ncm-ccm is guaranteed to only run after the initial ccm bootstrap so it's safe to put things in here which depend on other parts of the tree (such as ncm-accounts). 

So a more practical concern is that this makes it difficult to use the useful group_readable functionality on sites that don't use ncm-accounts. Can we make that possible somehow?